swap_meet/vendor.py

```diff
@@ -1,2 +1,72 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None) -> None:
+        self.inventory = inventory if inventory is not None else []
```

Comment:
👍  Great work in handling default parameters for mutable types. It's unsafe to set [] as the default value, but we can use `None` to stand in for the value when not supplied by the caller, and then use a new empty list in its place.